benchmark/csharp/Tpch/Tpch.csproj

```diff
@@ -11,7 +11,7 @@
   <ItemGroup>
     <!-- Workaround https://github.com/dotnet/project-system/issues/935 -->
     <None Include="**/*.cs" />
-    
+
```

Comment:
why this change?

also, you can remove net461 since it doesn't support intrinsics.

Reply:
@imback82 Will do!